apps/foundation-demo/src/app/misc/header/reducers/index.ts

```diff
@@ -0,0 +1,33 @@
+import { ActionReducerMap, createSelector, createFeatureSelector, MemoizedSelector } from '@ngrx/store';
+
+import * as fromFoundationSidebar from './sidebar.reducer';
+
+export interface FoundationHeaderState {
+  foundationSidebar: fromFoundationSidebar.State;
```

Comment:
I don't think header/sidebar should be related at all. There's no need for the connection other than visual appearances.

Reply:
done